app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -86,9 +86,14 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
 
     var sessionFeature: SessionFeature? = null
 
+    private lateinit var _pinnedTileViewModel: PinnedTileViewModel
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         initSession()
+
+        val factory = ViewModelFactory(context!!.serviceLocator)
+        _pinnedTileViewModel = ViewModelProviders.of(this, factory).get(PinnedTileViewModel::class.java)
```

Comment:
Optional nit: Since this isn't backing anything in this class, I'm not sure the underscore prefix applies here.  Kind of a grey area though.  Personally, I would take it off.